### PR TITLE
Add SkipLinks

### PR DESCRIPTION
**Pull request by jeffkamo**
SkipLinks are an accessibility feature that should be included on all of Mobify's project *by default*. This way, screen reader users, assistive technology and keyboard users are able to skip to relevant content much more easily.

## Changes
- Added SkipLink component to scaffold
- Added nightwatch tests that check to ensure the SkipLinks and their targets are valid (at least on the homepage)
- Make Merlin's Potions the default initial URL for local Nightwatch tests

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlins Potions](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.min.js&disabled=0&domain=&scope=1)
- (if testing in desktop Chrome) Ensure that dev tools is closed, and then...
    - Click the browser's URL bar
    - On your keyboard, press `tab` until you arrive on the page itself
    - Ensure that you are able to tab to the "Skip to x" links
    - Ensure pressing `enter` on any of the skip links sends you to that link's destination
- (if testing on device) Ensure VoiceOver is activated, and then...
    - Tap Safari's url bar
    - Swipe right ("tabbing") until you arrive on the page itself
    - Ensure that you are able to tab to the "Skip to x" links
    - Ensure double tapping the screen ("clicking") on any of the skip links sends you to that link's destination

### CODE REVIEW

File: app/containers/app/container.js [hunk 1]
```diff
@@ -15,22 +16,25 @@ class App extends React.Component {
         let currentTemplate = `t-${this.props.children.props.route.routeName}`
 
         return (
-            <div id="outer-container" className="t-app">
+            <div id="app" className="t-app">
```
Comment (jeffkamo) [inline app/containers/app/container.js]: I wouldn't mind confirmation that these IDs weren't named anything specific for a reason, and if these are safe to change?

File: app/components/skip-links/index.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            <a href="#app-navigation" className="c-skip-links__anchor">
+                Skip to main navigation
+            </a>
+
+            <a href="#app-footer" className="c-skip-links__anchor">
+                Skip to footer
+            </a>
+        </div>
+    )
+}
```
Comment (DanielBrookRoberge) [inline app/components/skip-links/index.jsx]: I guess by being in the scaffold it isn't meant to be a highly reusable component, but do we really want to remove the possibility of the user passing in additional `className` props?

File: app/containers/app/container.js [hunk 3]
```diff
@@ -15,22 +16,25 @@ class App extends React.Component {
...
                 <IconSprite />
+                <SkipLinks />
 
-                <main id="page-wrap" className={currentTemplate}>
```
Comment (jeffkamo) [inline app/containers/app/container.js]: The `main` element is often misused as an overall wrapper when it *should* in fact be used as a wrapper around only the relevant main content.

...In fact, I meant to re add this element in the the right place. I'll do that as well as include some other landmark roles for more a11y goodness.

**Comment by DanielBrookRoberge** [comment]
Also, there are lint errors.

File: app/components/skip-links/index.jsx (hunk 2 above)
Comment (jeffkamo) [inline app/components/skip-links/index.jsx]: I was actually just being lazy when I accidentally removed and and didn't add it back in. I'll re-include it now.

**Pull request merged**